lib/utils/types.ts

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+export interface Field {
+  root: string
+  minus: string
+  input: string
+  plus: string
+}
+
+export interface Image {
+  isPrimary: boolean
+  imageUrl: string
+  alt: string
+  Image: React.FC
+}
+
+export interface PricingClassNames {
+  promotional: string
+  original: string
+}
```

Comment:
Make sure to leave a blank line at the end of every file as best practice to comply with [POSIX standards](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206)